.github/CODEOWNERS

```diff
@@ -281,6 +281,10 @@
 # ServiceLabel: %Digital Twins %Service Attention
 /sdk/digitaltwins/                              @drwill-ms @timtay-microsoft @abhipsaMisra @vinagesh @azabbasi @bikamani @barustum @sourabhguha @inesk-vt @jamdavi
```

Comment:
For some reason, sourabh and ines are still here. Would you remove them for me please?

Reply:
oops, only fixed timeseriesinsights and missed digital twins.